### DISCUSSION TITLE
Updating FreeRDP

### DISCUSSION THREAD

**Discussion by Rutsche1243** [FreeRDP/FreeRDP#10655] [Q&A]
Hello all,

Im new to FreeRDP, and to Github.
I use an Apache Guacamole installation that used FreeRDP 2.6.1. 

I am looking to upgrade the version of FreeRDP so i can rebuild my Guacamole with FreeRDP Version 2.10.0
Is there a best way to do this correctly?

**Comment by hardening** [comment]
You'd better ask that to the guacamole project.

**Comment by akallabeth** [comment]
well, I assume you use some linux distribution?
-> check the distro paths to update packages
-> as @hardening said, ask guacamole project  if you got it directly from them
